extensions/cheetah/package.json

```diff
@@ -0,0 +1,132 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "version": "1.0.0",
+  "name": "cheetah",
+  "title": "Cheetah",
+  "description": "Search for a local Git project and open it with the specified application.",
+  "icon": "logo.png",
+  "author": "ronglecat",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "editor",
+      "title": "Open With Editor",
+      "description": "Open the specified project using the editor.",
+      "mode": "view",
+      "icon": "command/editor.png",
+      "keywords": [
+        "editor",
+        "Editor"
+      ]
+    },
+    {
+      "name": "git",
+      "title": "Open With Git GUI",
+      "description": "Open the specified project using the Git GUI.",
+      "mode": "view",
+      "icon": "command/git.png",
+      "keywords": [
+        "git",
+        "Git"
+      ]
+    },
+    {
+      "name": "terminal",
+      "title": "Open With Terminal",
+      "description": "Open the specified project using the terminal.",
+      "mode": "view",
+      "icon": "command/terminal.png",
+      "keywords": [
+        "terminal",
+        "Terminal"
+      ]
+    },
+    {
+      "name": "finder",
+      "title": "Open With Finder",
+      "description": "Open the specified project using the Finder.",
+      "mode": "view",
+      "icon": "command/finder.png",
+      "keywords": [
+        "finder",
+        "Finder"
+      ]
+    },
+    {
+      "name": "chooseApp",
+      "title": "Choose Application",
+      "description": "Specifying applications for projects.",
+      "mode": "view",
+      "icon": "command/app.png"
+    },
+    {
+      "name": "editCache",
+      "title": "Edit Cache File",
+      "description": "Open the cache file using the default editor.",
+      "mode": "no-view",
+      "icon": "command/editCache.png"
+    },
+    {
+      "name": "clearCache",
+      "title": "Clear Cache",
+      "description": "Clear cache files.",
+      "mode": "no-view",
+      "icon": "command/clearCache.png"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "workspaces",
+      "description": "The tool will search for projects in the working directory you set, with multiple directories separated by commas, Need to configure the absolute path of the directory, This directory should contain projects managed with Git.",
```

Comment:
Why not support path like `~/Documents`

Reply:
Originally planned to be supported in subsequent iterations, this feature has now been added, and README.md has been modified